encoding/toml_test.ts

```diff
@@ -480,3 +484,29 @@ Deno.test({
     assertEquals(actual, expected);
   },
 });
+
+Deno.test({
+  name: "[TOML] Single-line string comment error",
```

Comment:
This test passes when running against the previous version of the `encoding/toml.ts` code -- is that expected?

Reply:
This test was added to add coverage for this line : https://github.com/denoland/deno_std/blob/635da0f56c8c7b2294b0448763bda1d055dcfe44/encoding/toml.ts#L101